org.opent2t.sample.hub.superpopular/org.opent2t.sample.hub.superpopular.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.opent2t.sample.hub.superpopular">
+    <description>OpenT2T SuperPopular Hub interface</description>
+
+    <property name="name" type="s" access="read">
+     <description>ReadOnly, Gets the name of the hub.</description>
+    </property>
+    
+    <property name="devices" type="a" access="read">
```

Comment:
>a [](start = 35, length = 1)

Just "a" by itself isn't a valid AllJoyn type code. It has to be an array of *something*.